run/calling_dash.py

```diff
@@ -170,34 +179,6 @@ def main():
     ]
 
     # Main content of streamlit app
-    st.write(f"## Patient {st.session_state['name_idx'] + 1}/{len(patient_ids)}")
-    col1, col2, col3 = st.columns(3)
-    with col1:
-        st.button(
-            "Vorige patient",
-            on_click=navigate_patients,
-            args=(len(patient_ids), False, current_response.call_status),
-        )
-    with col2:
-        with st.popover(
-            "🔍 Zoeken", help="Zoek op telefoonnummer om een patient te vinden."
-        ):
-            phone_number = st.text_input("Zoek op telefoonnummer...")
-            st.button(
-                "Zoek",
-                on_click=search_number,
-                args=(Session, phone_number, patient_ids),
-            )
-    with col3:
-        st.button(
-            "Volgende patient",
-            on_click=navigate_patients,
-            args=(len(patient_ids), True, current_response.call_status),
-        )
-    st.header("Patient-gegevens")
-    if enable_dev_mode:
-        st.write(f"- ID: {patient_ids[st.session_state['name_idx']]}")
-
     render_patient_info(
         Session,
         current_response,
```

Comment:
While we're add it can we also create a render afspraakoverzicht function? That way all UI is in separate functions